hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -172,6 +174,9 @@ public class ServerManager {
   /** Listeners that are called on server events. */
   private List<ServerListener> listeners = new CopyOnWriteArrayList<>();
 
+  /** Configured value of HConstants.REJECT_DECOMMISSIONED_HOSTS_KEY */
+  private boolean rejectDecommissionedHostsConfig;
```

Comment:
Use the `volatile` keyword on this field and omit the `synchronized` block below.

Reply:
Fixed.